tools/serial_rf_spy.py

```diff
@@ -16,7 +16,8 @@ def __init__(self, serial_port, rtscts=None):
     if not rtscts:
       rtscts = int(os.getenv('RFSPY_RTSCTS', 1))
 
-    self.ser = serial.Serial(serial_port, 19200, rtscts=rtscts, timeout=1)
+    # self.ser = serial.Serial(serial_port, 19200, rtscts=rtscts, timeout=1)
+    self.ser = serial.Serial(serial_port, 115200, timeout=1)
```

Comment:
Is this intentional (19200 -> 115200) ?  I think it will not work with other serial implementations.